docs/guides/debugging.md

```diff
@@ -217,6 +217,10 @@ Then when we re-run our test we get:
 
 With the use of the interactive debugger; we were able to examine the state of the embedded containerized 
 
+## Demo
+
+[![asciicast](https://asciinema.org/a/Q6dKRpnuyl7IpnnF4e3fHhYoi.svg)](https://asciinema.org/a/Q6dKRpnuyl7IpnnF4e3fHhYoi)
```

Comment:
Remember to add speed=2

Reply:
Could you also adjust the title of the video to something meaningful?

Reply:
done and done :)